Makefile

```
@@ -20,17 +49,109 @@ meson:
-	ctest --test-dir build-release
+markdownlint:
+	@files="$(shell git ls-files '*.md')"; if [ -z "$$files" ]; then echo "markdownlint: no markdown files found"; else $(MARKDOWNLINT) $(MARKDOWNLINT_ARGS) $$files; fi
 
-test-meson: meson
-	meson test -C meson-debug
-	meson test -C meson-release
+activity-validate:
+	./tools/lint/validate_activity_log.sh
 
-test-both: test-cmake test-meson
+log:
```

checkmake (0.2.2)

[warning] Target body for "log" exceeds allowed length of 5 (21).

(maxbodylength)